domains/superskibidi

```diff
@@ -0,0 +1,11 @@
+{
+  "owner": {
+    "username": "3TERN4LF1R3",
+    "discord": "3tern4l_f1r3",
+    "email": ""
+  },
+
+  "record": {
+    "CNAME": "3TERN4LF1R3.github.io"
```

Comment:
Your site is a 404 page and I don't see any repository on your profile which has the contents of the picture